filewatcher/filewatcher_test.go

```diff
@@ -375,7 +400,10 @@ func TestParserSizeLimit(t *testing.T) {
 	}()
 	// Give it some time to handle the file content change
 	time.Sleep(time.Millisecond * 500)
-	compareBytesData(t, data.Get(), expectedPayload2)
+	// We expect the second parse would fail because of the data is beyond the
+	// limit, so the data should still be expectedPayload1
+	compareBytesData(t, data.Get(), expectedPayload1)
+	wrapper.check(1)
```

Comment:
🔕 return the value and compare in the test

Reply:
that would require us to compare the logged text in a test, so changing the text (or the error message) in the future would break the test.

Reply:
I mean the number (i.e. 1)